src/lib/functions/form-submissions-handler.js

```diff
@@ -126,6 +126,12 @@ const createFormSubmissionHandler = function ({ functionsRegistry, siteUrl }) {
   }
 }
 
+// note: use Object.fromEntries post Nodejs 12.0.0
+const getObjectFromParams = (urlParams) => {
+  const entries = [...urlParams]
+  return entries.reduce((prev, [key, value]) => ({ ...prev, [key]: value }), {})
+}
+
```

Comment:
NIT: Maybe you could mark the function as deprecated as there is already an internal issue to drop the support of Nodejs < 12.0.0

```suggestion
/**
 * Converts URLSearchParams to an object
 * @deprecated can be replaced with `Object.fromEntries` once the support for older Node.js Versions than `12.0.0` is dropped.
 * @param {URLSearchParams} urlParams 
 * @returns {Record<string, any>}
 */
?**
const getObjectFromParams = (urlParams) => {
  const entries = [...urlParams]
  return entries.reduce((prev, [key, value]) => ({ ...prev, [key]: value }), {})
}

```

Reply:
I've added the suggestion.